examples/bedrock-claude-factual-knowledge.ipynb

```diff
@@ -293,14 +272,12 @@
     },
     "pycharm": {
      "name": "#%%\n"
-    }
+    },
+    "tags": []
    },
    "outputs": [],
    "source": [
-    "#\n",
-    "# Pretty-print the evalaution output (notice the score).\n",
-    "#\n",
-    "\n",
+    "# Pretty-print the evaluation output (notice the score).\n",
```

Comment:
Non-blocking comment: I find `mercury` looks much better within notebooks for printing out JSON type objects.

Reply:
Will look into using `mercury` at a later point. Thanks for the tip!